AngularJS/experts_to_follow.md

```diff
@@ -0,0 +1,3 @@
+Name | Link
+------------ | ------------- 
+Todd Motto | https://twitter.com/toddmotto
```

Comment:
Also add other social media links (if available). For example, https://github.com/GDGAhmedabad/Learning-Resources/blob/master/android/experts_to_follow.md